<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaDescargarCertificado</class>
 <widget class="QDialog" name="VistaDescargarCertificado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Descargar Certificado</string>
  </property>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCedula">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCursos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDescargar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Descargar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCedula">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CÃ©dula estudiante</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
